### DISCUSSION TITLE
Admin access

### DISCUSSION THREAD

**Discussion by drewington** [mbround18/valheim-docker#151] [Q&A] [answered]
Hey All,

Had some googles and looked all over on this repo. I didn't see much on how to configure the adminlist.txt file, or how to gain admin access using this docker? Any help is appreciated, thanks.

**Comment by mbround18** [comment, marked as answer]
Best way to access them right now is by using the cli tool called nano inside the container or from the host :)

I personally pair this docker container with 
https://hub.docker.com/r/hurlenko/filebrowser

To manage files by using the same PUID and GUID

**Reply by Akibby** [reply]
I've been having trouble accessing admin privileges too. I have verified my name is in the adminlist.txt file in the container, but I still am not admin when I try to do things in the server. One question I have is if I should be using my steam username or my character name in the file. I have both right now but neither works.

**Reply by mbround18** [reply]
> I've been having trouble accessing admin privileges too. I have verified my name is in the adminlist.txt file in the container, but I still am not admin when I try to do things in the server. One question I have is if I should be using my steam username or my character name in the file. I have both right now but neither works.

You should have your steam 64 ID inside the adminlist.txt

Go to https://steamid.io/ and type your steam username in. When it pops up copy the 64 ID from there and put it into the adminlist with one per line.

Alternatively inside of Valheim you can hit F2 to see your players name and your steam 64 ID. That's what you want to put in the txt file.

**Reply by drewington** [reply]
> Best way to access them right now is by using the cli tool called nano inside the container or from the host :)
> 
> I personally pair this docker container with
> https://hub.docker.com/r/hurlenko/filebrowser
> 
> To manage files by using the same PUID and GUID

Ah never heard of this filebrowser docker, brilliant! I will give it ago. I appreciate all the help.

If that doesn't work. The file location in the container would be the following? (as Mikegreen pointed out)

/home/steam/.config/unity3d/IronGate/Valheim

**Comment by mikegreen** [comment]
You can see those in the volume you config in docker-compose
ie, would be in this saves folder: 
**/opt/docker-configs/valheim-server/valheim/saves**:/home/steam/.config/unity3d/IronGate/Valheim

```$ ll /opt/docker-configs/valheim-server/valheim/saves/
total 28
drwxr-xr-x 3 dockeruser dockeruser 4096 Feb 21 20:11 ./
drwxrwxr-x 4 dockeruser dockeruser 4096 Feb 21 20:10 ../
-rw-r----- 1 dockeruser dockeruser   39 Feb 21 20:11 adminlist.txt
-rw-r----- 1 dockeruser dockeruser   40 Feb 21 20:11 bannedlist.txt
-rw-r----- 1 dockeruser dockeruser   42 Feb 21 20:11 permittedlist.txt
-rw-r----- 1 dockeruser dockeruser  218 Feb 21 21:04 prefs
drwxr-x--- 2 dockeruser dockeruser 4096 Feb 21 21:24 worlds/```
